ToDoApp/ToDoApp/ContentView.swift

```diff
@@ -6,7 +6,6 @@
 //
 
 import SwiftUI
-
```

Comment:
it is good practice to not make changes which are not related to a PR. for example, this new line. since a bunch of such changes may keep a reviewer unfocused